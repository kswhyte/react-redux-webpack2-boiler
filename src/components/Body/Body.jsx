import React from 'react';
import PropTypes from 'prop-types';

const Body = (props) => {

  Body.propTypes = {

  };

    return (
        <div className="body-wrapper">
            <h1>Body</h1>
        </div>
    );
};

export default Body;
