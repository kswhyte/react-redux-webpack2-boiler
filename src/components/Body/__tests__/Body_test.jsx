import React from 'react';
// import { storiesOf, action, linkTo } from '@kadira/storybook';
import Body from '../Body';

// import { specs, describe, it } from 'storybook-addon-specifications';

import {storiesOf, describe, it, specs
} from "../../../../.storybook/facade";

import {shallow} from 'enzyme';
import expect from 'expect';


const stories = storiesOf('Body', module);

stories.add('Story', () => {
    const bodyStory = (
      <Body/>);

    specs(() => describe('Show a successful alert', () => {
        it('Should render the Body component without crashing', () => {
            let output = shallow(bodyStory);
            expect(output.find('.body-wrapper').length).toEqual(1);
        });
    }));
    return bodyStory;
});
