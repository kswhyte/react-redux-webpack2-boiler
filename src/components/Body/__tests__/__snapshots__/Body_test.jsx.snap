// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Story 1`] = `
<div
  className="body-wrapper"
>
  <h1>
    Body
  </h1>
</div>
`;
